src/main/java/net/masterthought/cucumber/util/Status.java

```diff
@@ -13,7 +13,8 @@
     SKIPPED("#88AAFF"),
     PENDING("#FBB907"),
     UNDEFINED("#FBB957"),
-    MISSING("#FBB9A7");
+    MISSING("#FBB9A7"),
+    HIDDEN("");
```

Comment:
At the moment the Java code contains logic and view concerns mixed together - long term I would like to separate these out so the Java code is view / display agnostic 

Sent from my iPhone

> On 13 May 2015, at 20:19, Damian Szczepanik <notifications@github.com> wrote:
> 
> In src/main/java/net/masterthought/cucumber/util/Status.java:
> 
> > @@ -13,7 +13,8 @@
> >      SKIPPED("#88AAFF"),
> >      PENDING("#FBB907"),
> >      UNDEFINED("#FBB957"),
> > -    MISSING("#FBB9A7");
> > +    MISSING("#FBB9A7"),
> > +    HIDDEN("");
> check with isBackground method so I guess you can make it similar but for sure not here
> 
> —
> Reply to this email directly or view it on GitHub.
>